Stripe/STPCard.m

```diff
@@ -84,19 +87,34 @@ + (NSString *)stringFromBrand:(STPCardBrand)brand {
     }
 }
 
+#pragma mark - STPCardFundingType
+
++ (NSDictionary <NSString *, NSNumber *> *)stringToFundingMapping {
+    return @{
+             @"credit": @(STPCardFundingTypeCredit),
+             @"debit": @(STPCardFundingTypeDebit),
+             @"prepaid": @(STPCardFundingTypePrepaid),
+             @"other": @(STPCardFundingTypeOther),
```

Comment:
Cleaned up in https://github.com/stripe/stripe-ios/pull/711/commits/2093f279c4b9f47ce1b84ce86d585362bf7ba0cd